src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1184,6 +1185,10 @@ void PolicyHandler::OnAllowSDLFunctionalityNotification(
             accessor.GetData().end(),
             DeactivateApplication(device_handle,
                                   application_manager_.state_controller()));
+#ifdef SDL_REMOTE_CONTROL
```

Comment:
@dev-gh I think that such functionality should not be under ifdef. 
SDL_REMOTE_CONTROL is actually for remote control but not for plugins support

Reply:
@dev-gh ?

Reply:
@LuxoftAKutsan 
Code under ifdef requires access to functional_module header which is also under same ifdef in CMakefile